test/validators.mbtiles.test.js

```diff
@@ -111,6 +111,7 @@ test('lib.validators.mbtiles: file too big', function(t) {
     t.ok(err, 'expected error');
     t.equal(err.code, 'EINVALID', 'expected error code');
     t.equal(err.message, expected.mbtilesErrors.filetoobig, 'expected error message');
+    console.log('made it here?');
```

Comment:
Funny story - we did make it, but then the tilelive-copy operation kept flying right past it.

Ryan, have you ever heard of the `tilelive.copy` callback being called _before_ the copy operation is finished?